### PR TITLE
corect info for cephvolumenumber variable

### PR DESCRIPTION
**Pull request by mimi1vx**

### CODE REVIEW

File: scripts/mkcloud [hunk 1]
```diff
@@ -1220,8 +1220,9 @@ Optional
     TESTHEAD='' | 1  (default='')
         use Media from Devel:Cloud:Staging and add test update repositories (except for GM* targets)
     cephvolumenumber  (default=0)
-        the number of 11GB ceph volumes that will be created per node
-        note: proposal step does NOT contain a ceph proposal, do it manually
+        the number of 21GB ceph volumes that will be created per node
```
Comment (aspiers) [inline scripts/mkcloud]: Not quite right - it's 21GB by default but can be changed by setting `cephvolume_hdd_size`.

File: scripts/mkcloud [hunk 2]
```diff
@@ -1220,8 +1220,9 @@ Optional
...
+        note: proposal step does contain a ceph proposal only in SUSE CLOUD 4, in
```
Comment (aspiers) [inline scripts/mkcloud]: Nitpick: should be "SUSE Cloud"

**Comment by aspiers** [comment]
:+1: thanks!

**Comment by jdsn** [comment]
:+1:

**Pull request merged**